docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,225 @@
+
+**More Information**: You can find more information about the [Org Browser](https://developer.salesforce.com/tools/vscode/en/user-guide/org-browser) in the Salesforce Extension Pack documentation.
+
+![org_browser_small](../../../images/org_browser_small.jpg)
+
+
+Click Org Browser and scroll down to **Custom Objects**. The first object you see is Account. You can view the metadata of the **Account** object (and do a few more cool things, but we’ll get to those in a bit). 
+
+## Use SOQL Builder to Build and Run your First Query
+
+Code Builder comes preloaded with everything you need to build and run a SOQL query. 
+Let’s build and run a simple SOQL query on the Account object:
+
+1. Press Ctrl+Shift+P (⇧⌘P)  to view the Command Palette.
+2. Type “SOQL” in the search box and then select **SFDX: Create Query in SOQL Builder**.
+3. Select  the following:
+
+    a. **Account** Object in **From** field.
+
+    b. **BillingState** and **NumberofEmployees** in **Fields**.
+
+    c. Press **Run Query** to run this simple SOQL Query on your org.
+
+You’re prompted to save your changes if you close the untitled file. 
+5. Save the changes to a file using a “.soql” extension.
+6. Let’s rerun the saved query – Right click the file and select **Open With…** then select SOQL Builder.
+The file opens in SOQL Builder and you can rerun or edit the query as you wish.
+
+**More Information**: For more information on building complex queries see the [SOQL Builder](https://developer.salesforce.com/tools/vscode/en/soql/soql-builder) in the Salesforce Extensions for Visual Studio Code documentation.
+
+## Connect to a Different Org
+During the course of development, you'll use different types of orgs for different stages. For example, it's common to use a Developer sandbox or Development Edition org during the development phase, and move to other sandbox types for integration, testing, and staging. Eventually, you'll deploy your changes to a production org. You can connect Code Builder to any of these orgs to deploy or retrieve metadata.
+
+To log into another org:
+
+1. Click the org picker(which show the alias for the current org) in the status bar, to  bring  up the command palette.
+2. From the Command Palette run **SFDX: Authorize an Org**.
+3. Enter the login URL or select the org you want to log into.
+4. Enter an alias for the org, for example, dev_pro_sandbox or my_playground.
+5. A code is displayed in a text box. click **Connect**.
+6. Log in with the relevant username and password. Click **Allow**. 
+7. Click **Continue**. You’re now connected to a different org, and its name is visible in the status bar at the bottom.
+
+Once you authorize an org, we take care of future authorizations so you don't have to continually log in. Just click the org’s name and then choose the org from the list.	
```

Comment:
```suggestion
After you authorize an org, we take care of future authorizations so you don't have to continually log in. Just click the org’s name and then choose the org from the list.	
```

Reply:
Our style guide says to use "after" vs. "once" - which I totally have to self-edit for.